package.json

```diff
@@ -1,7 +1,6 @@
 {
   "license": "UNLICENSED",
   "devDependencies": {
-    "@outreach/semantic-release-circleci-orb": "^1.1.9",
```

Comment:
release failed due to missing semantic-release-circleci-orb - let's add them back